ingress-node-firewall/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the ingress-nodefw v1alpha1 API group
+//+kubebuilder:object:generate=true
+//+groupName=ingress-nodefw.ingress-nodefw
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "ingress-nodefw.ingress-nodefw", Version: "v1alpha1"}
```

Comment:
Should the group not be an FQDN with a valid TLD?

Reply:
not sure, I checked some of the operators we ahve in OCP and it seems free form , can u show me an example of what u want to be done ?

Reply:
`ingress.node.firewall.openshift.io` is good ?

Reply:
I think `ingressnodefirewall.openshift.io`. Using dots to separate "ingress node firewall" may erroneously tell the user there are multiple subdomains instead of one.